scripts/nb-tester/notebooks.toml

```diff
@@ -51,8 +80,10 @@ notebooks_that_submit_jobs = [
 # A job is "too big" if a cell can't run in under 5 mins, or we run out of
 # memory on a reasonable device.
 notebooks_no_mock = [
-    "docs/start/hello-world.ipynb",
+    "docs/guides/hello-world.ipynb",
```

Comment:
Thanks for the review! Yes, both should exist for the moment until we make the migration. I moved the `docs/start/hello-world.ipynb` entry to the end of the list (line 88) to clarify that the comment only affects that line, just in case.